recipes-core/packagegroups/packagegroup-ni-coreimagerepo.bb

```diff
@@ -21,6 +21,9 @@ RDEPENDS_${PN} = "\
 "
 
 RDEPENDS_${PN}_append_x64 = "\
-	${@bb.utils.contains('DISTRO_FEATURES', 'x11', 'packagegroup-ni-xfce', '', d)} \
+	${@bb.utils.contains('DISTRO_FEATURES', 'x11', '\
+	packagegroup-ni-xfce \
+	qtbase \
```

Comment:
`packagegroup-ni-coreimagerepo` isn't used in sumo. It was the core feed group for the deprecated `nilrt-nxg` distro flavor, but isn't invoked in our sumo CG components.

You'll need to add your deps here to the [`ADDITIONAL_CORE_FEED_RECIPES`](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=/src/rtos/rtoslegacyd/NIOpenEmbedded/x64_dkms.mak&version=GBmain&line=11&lineEnd=12&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) variable in the `NIOpenEmbedded/x64_dkms.mak` target.

Reply:
Removed the commit from here.